ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml

```diff
@@ -124,13 +124,14 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: rosa-aws-sts-hypershift
-- as: aws-rosa-sts-hypershift-prod-guest-p1-f7
-  cron: 8 3 2,10,17,24 * *
+- as: aws-rosa-sts-hypershift-prod-sector-guest-p1-f7
```

Comment:
The test name is updated, but diff from the suggested one. Would you mind to ack if this is expected, or update accordingly if it's not ?

Reply:
@heliubj18 We should better change to `aws-rosa-sts-hypershift-sector-prod-guest-p1-f7`